ios/iosHackthonApp/Scene Delegate/SceneDelegate.swift

```diff
@@ -11,9 +11,13 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
   func scene(_ scene: UIScene, willConnectTo session: UISceneSession, options connectionOptions: UIScene.ConnectionOptions) {
 
     let loginView = LoginView(viewModel: loginViewModel)
+    let appState = AppState()
+
     if let windowScene = scene as? UIWindowScene {
       let window = UIWindow(windowScene: windowScene)
-      window.rootViewController = UIHostingController(rootView: loginView)
+      window.rootViewController = UIHostingController(rootView:
+                                                        loginView
+                                                        .environmentObject(appState))
```

Comment:
This looks a little strange to me. I don't know if it is possible but could we do the `.environmentObject` on line 13?

Reply:
Yes changed this.